.tekton/basic-component-pipeline.yaml

```
@@ -174,42 +155,42 @@ spec:
       operator: in
       values: [ "true" ]
     workspaces:
-    - name: output
-      workspace: workspace
     - name: basic-auth
       workspace: git-auth
 
   - name: determine-image-tag
     params:
     - name: tag-suffix
       value: $(params.output-tag-suffix)
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
     runAfter:
     # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
```

I think, the comment now should be placed closer to `SOURCE_ARTIFACT`.
```suggestion
    # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
    - name: SOURCE_ARTIFACT
      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
    runAfter:
```

Or even better, it can now be moved to param description in the definition of `determine-image-tag` itself. We simply want to say that source artifact should be result of git clone, not prefetch because prefetch would fail as dirty.